docs/ref/files/file.txt

```diff
@@ -91,6 +91,10 @@ The ``File`` Class
     ``read``, ``readinto``, ``readlines``, ``seek``, ``softspace``, ``tell``,
     ``truncate``, ``writelines``, ``xreadlines``.
 
+        .. versionchanged:: 1.9
```

Comment:
deindent this by 4 spaces
Also the note should be "self-contained" as described in https://docs.djangoproject.com/en/1.8/internals/contributing/writing-documentation/#documenting-new-features